views/jira-configuration-new.hbs

```diff
@@ -0,0 +1,253 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="ap-local-base-url" content="{{localBaseUrl}}" />
+    <meta name="public-url" content="{{APP_URL}}" />
+    <title>{{title}}</title>
+    <link
+      rel="stylesheet"
+      href="/public/aui/aui-prototyping.css"
+      integrity="DTM1Q+8lU7SzJT+FWr0JFisCSZlwfM0GiAKYy7h1s9vIKa/CIh37s9NuOCqIOgK4tmqrjLK4NuWuIPUQNsikHA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="/public/css-reset/bundle.css" media="all" />
+    <link
+      rel="stylesheet"
+      href="/public/atlassian-ui-kit/bundle.css"
+      media="all"
+    />
+    <link rel="stylesheet" href="/public/css/global.css" media="all" />
+    <link
+      rel="stylesheet"
+      href="/public/css/jira-configuration-new.css"
+      media="all"
+    />
+    <script src="/public/js/jquery.min.js" nonce="{{nonce}}"></script>
+  </head>
+
+  <body class="aui-page-hybrid">
+    <div class="jiraConfiguration">
+      <header class="jiraConfiguration__header">
+        <h1 class="jiraConfiguration__header__title">GitHub configuration</h1>
+
+        {{#if hasConnections}}
+          <button
+              class="jiraConfiguration__header__title__cta ak-button ak-button__appearance-primary aui-button aui-dropdown2-trigger"
+              aria-controls="connectBtnDropdown"
+          >Connect GitHub organization
+          </button>
+          <aui-dropdown-menu id="connectBtnDropdown">
+            <aui-section>
+              <a class="jiraConfiguration__connectBtnDropdownOption add-organization-link"
+                 target="_blank">
+                <span>Connect GitHub Cloud</span>
+                <i class="aui-icon aui-iconfont-question-filled"></i>
+                <div class="jiraConfiguration__connectGHCloudTooltip">Select this option if the URL of your GitHub
+                  organization starts with "http://github.com".
+                  This includes “Free”, “Team” and “GitHub Cloud Enterprise” accounts.
+                </div>
+              </a>
+              <a class="jiraConfiguration__connectBtnDropdownOption select-github-version-link"
+                 target="_blank">
```

Comment:
We shouldn't have target blank here. We need it for cloud because that involves navigating _out_ of Jira and 'into' GH. For server, we'll be navigating to a screen that's still on the Jira side of the flow e.g. jira/ghe-server-list (or whatever we end up calling that path). This should happen in the same tab.